tools/glide-pindeps/glide/pin.go

```diff
@@ -0,0 +1,42 @@
+package glide
+
+import (
+	"fmt"
+	"os"
+)
+
+// PinDependencies receives a LockFile and a YamlFile and updates or inserts
+// missing imports onto the YamlFile. Dependencies with a non-empty "repo" field
+// are skipped, as they are assumed to contain openshift-specific dependencies.
+func PinDependencies(lockfile *LockFile, yamlfile *YamlFile) error {
+	if lockfile == nil || yamlfile == nil {
+		return fmt.Errorf("both a lockfile and a yaml file are required")
+	}
+
+	for _, lockDep := range lockfile.Imports {
+		if len(lockDep.Repo) > 0 {
+			fmt.Fprintf(os.Stderr, "info: skipping package with %q field: %s\n", "repo", lockDep.Name)
```

Comment:
substitution looks weird.

Reply:
maybe something more like: `skipping: lock file "name" points to fork "repo fileld"`